pulsar-broker/src/main/java/org/apache/pulsar/broker/delayed/bucket/BucketDelayedDeliveryTracker.java

```diff
@@ -306,7 +308,19 @@ private synchronized CompletableFuture<Void> asyncMergeBucketSnapshot() {
                 minIndex = i;
             }
         }
-        return asyncMergeBucketSnapshot(values.get(minIndex), values.get(minIndex + 1));
+        ImmutableBucket immutableBucketA = values.get(minIndex);
+        ImmutableBucket immutableBucketB = values.get(minIndex + 1);
+        log.info("[{}] Merging bucket snapshot, bucketAKey: {}, bucketBKey: {}", dispatcher.getName(),
```

Comment:
We should change to debug level since it will make a lot of noise if you have many topics on the broker.

Reply:
Ok.